src/modules/mentorships/dto/userSummary.dto.ts

```diff
@@ -0,0 +1,10 @@
+export class UserSummaryDto {
```

Comment:
I think we should use the existing `User` [DTO](https://github.com/Coding-Coach/find-a-mentor-api/blob/master/src/modules/common/dto/user.dto.ts) that we have rather than creating a new one 🤔

Reply:
The user DTO has some additional which wasn't mentioned on the issue. Is that still ok to include it?

Reply:
How about using [`PickType`](https://docs.nestjs.com/openapi/mapped-types#pick)?

Reply:
I'm not 100% sure if the undefined properties will be returned, but let's try that. If we are getting undefined fields, let's use what @arku recommended.

Reply:
Updated to use `UserDto`. As you expected @crysfel this returned only the populated properties 👍